vitals/BMS.java

```diff
@@ -0,0 +1,21 @@
+package vitals;
+
+public class BMS {
```

Comment:
What Does BMS stands for

Reply:
BMS-BatteryManagementSystem,class is renamed to BatteryManagementSystem